### PR TITLE
Cattail jelly closes  #24414

### LINKED ISSUE #24414: Cattail Jelly: antiseptic, styptic, analgesic.
Near the base of the stalk of cattails, when peeling the inner leaves off to reveal the tender edible shoot inside, there is a sticky layer of cattail jelly between each inner leaf, in a surprisingly copious volume per plant. This substance is well-known amongst survivalists and historically as an antiseptic, strong analgesic, and styptic; applied to wounds directly or ingested. 

Mechanically, I'd say between 0-3 units of cattail jelly should be able to be harvested from cattails, with the chance of getting more increasing with survival skill. By volume it should use alcohol's units, such that 7 units are equivalent to 250ml, and it should be a byproduct of processing cattail stalks. It should be refinable into hemostatic powder, used in crafting bandages, as well as applicable raw as a painkiller; while there are plenty of sources of this clam, none actually note what the active ingredient is, so for game balance I'd err on the side of caution and leave it about the same strength as aspirin.

Sources: https://books.google.ca/books?id=2LB4OQLv-w8C&pg=PA223&lpg=PA223&dq=cattail+jelly+antiseptic&source=bl&ots=5C2CFfAPc-&sig=3ASO-2Yso-1I8P3sKQBm5qf9SXU&hl=en&sa=X&ved=0ahUKEwjE5d3u6rDcAhW-CjQIHfa3D3IQ6AEI3wEwGw#v=onepage&q=cattail%20jelly%20antiseptic&f=false & https://www.youtube.com/watch?v=eBfIORVLNUA amonst others easily available online.

### PR DESCRIPTION
**Pull request by vasekhodina**
#### Summary

```SUMMARY: Content "Adds new item 'Cattail Jelly' and recipe for crafting it"```

#### Purpose of change

Fixes #24414 - new medical item Cattail Jelly that has painkiller effect as well as disinfectant effect as
requested in the Issue.

#### Describe the solution

Cattail jelly can be crafted from cattail stalks by cutting them. Four cattail stalks produces 7 
batches of cattail jelly that can be applied by the heal action (same as other disinfectants).
Disinfectant strength is roughly the same as makeshift disinfectant.
Using it also adds painkiller effect (aspirin level).

I've chosen the crafting procedure as the cattail jelly is liquid and harvesting it from cattails 
drops it on ground automatically. It cannot be picked afterwards. Survival skill of 1 is required
as well as learning the recipe from any survival book. I think this is not something that would
just pop into your mind as you practice survival skill.

#### Describe alternatives you've considered

I've considered that cattail jelly would be harvestable from cattail, but due to the issue above left
the idea.
I've also considered that cattail jelly would be consumable as painkiller only, but that disables it's 
use as a disinfectant.

### CODE REVIEW

File: data/json/items/comestibles/med.json [hunk 1]
```diff
@@ -1372,5 +1372,29 @@
     "healthy": 25,
     "use_action": "PANACEA",
     "flags": "NPC_SAFE"
+  },
+  {
+    "id": "cattail_jelly",
+    "type": "COMESTIBLE",
+    "comestible_type": "MED",
+    "name": "Cattail Jelly",
```
Comment (Night-Pryanik) [inline data/json/items/comestibles/med.json]:
```suggestion
    "name": "cattail jelly",
```

Comment (Night-Pryanik) [reply data/json/items/comestibles/med.json]: Also add a plural name.

**Comment by Night-Pryanik** [comment]
Please run your changes through the web linter - http://dev.narc.ro/cataclysm/format.html.

File: data/json/items/comestibles/med.json [hunk 2]
```diff
@@ -1372,5 +1372,30 @@
     "healthy": 25,
     "use_action": "PANACEA",
     "flags": "NPC_SAFE"
+  },
+  {
+    "id": "cattail_jelly",
+    "type": "COMESTIBLE",
+    "comestible_type": "MED",
+    "name": "cattail jelly",
+    "name_plural": "cattail_jellies",
```
Comment (Night-Pryanik) [inline data/json/items/comestibles/med.json]:
```suggestion
    "name_plural": "cattail jellies",
```

**Comment by vasekhodina** [comment]
I've ran the web linter and pasted the output back to the file. Also fixed the underscore typo in name_plural.

**Comment by KorGgenT** [comment]
medical.json needs to be run through the web linter.

**Pull request merged**